src/examples/forms/input-group/ButtonPlacement.tsx

```diff
@@ -13,10 +13,6 @@ import { Row, Col } from '@zendeskgarden/react-grid';
 import { Dropdown, Trigger, Menu, Item } from '@zendeskgarden/react-dropdowns';
 import { ReactComponent as ChevronDownStroke } from '@zendeskgarden/svg-icons/src/16/chevron-down-stroke.svg';
 
-/**
- * The `react-dropdown` package includes a hidden input with the `Trigger` element.
- * This element must be positioned outside of the `InputGroup`.
- */
```

Comment:
this comment doesn't appear to be accurate. The hidden input is currently positioned _inside_ the `InputGroup` as expected.

Reply:
Weird, this must have been applicable before the v8 upgrade and the a11y improvements in Trigger.